Mithril update 2024-10-23 (#491)
IntersectMBO/cardano-updates: blog/2024-10-23-mithril.md

---
title: Mithril Team Update
slug: 2024-10-23-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team released the new distribution [`2442.0`](https://github.com/input-output-hk/mithril/releases/tag/2442.0). This release includes stable support for decentralized signature orchestration in both the signer and aggregator, stable support for the Cardano transaction client library, CLI and WASM, stable support for the Pythagoras Mithril era, along with bug fixes and performance improvements.
> 
> The team also continued implementing usage metrics with Prometheus endpoint and a Grafana dashboard for the aggregator. Additionally, the team completed the preparation of the Pythagoras Mithril era and the refactoring of the node logs to improve their readability.
> 
> Finally, they kept implementing optimizations in the CI and they updated the Decentralized Message Queue CIP pull request which has been granted the CIP number `CIP-0137`,

Reviewer: ```suggestion
This week, the Mithril team released the new distribution [`2442.0`](https://github.com/input-output-hk/mithril/releases/tag/2442.0). This release offers stable support for decentralized signature orchestration in both the signer and aggregator, the Cardano transaction client library, CLI and WASM, and the Pythagoras Mithril era. It also includes bug fixes and performance improvements.

The team also continued implementing usage metrics with a Prometheus endpoint and a Grafana dashboard for the aggregator. Additionally, they completed preparing the Pythagoras Mithril era and refactoring the node logs to improve their readability.

Finally, they kept implementing optimizations in the CI and updated the decentralized message queue CIP pull request, which has been granted the CIP number `CIP-0137`.
```
